internal/metamorph/store/postgresql/postgres.go

```diff
@@ -580,12 +583,17 @@ func (p *PostgreSQL) UpdateStatusBulk(ctx context.Context, updates []store.Updat
 		UPDATE metamorph.transactions
 			SET
 			status=bulk_query.status,
-			reject_reason=bulk_query.reject_reason
+			reject_reason=bulk_query.reject_reason,
+			last_modified=$1,
+			status_history=status_history || json_build_object(
+				'status', metamorph.transactions.status,
```

Comment:
Shouldn't be:
```
'status', bulk_query.status,
'timestamp', $1
```
?

Reply:
No, we want to add status with the start date of the status not the end one, this new last_modified date will be used for the next status change